services/appmaker/controller.go

```diff
@@ -32,12 +32,26 @@ func (s *server) Create(ctx context.Context, body *pb.RequestBody) (*pb.Response
 	if err != nil {
 		return nil, err
 	}
+	
+	filter := types.M{
+		"owner": body.GetOwner(),
+		"instance_type": "application",
```

Comment:
Create default constant for application instance type or database instance

Reply:
done